marketpulse/main/templates/fxosprice_all.html

```diff
@@ -65,7 +65,10 @@ <h3 class="title">Contributions</h3>
               {% endif %}
             </td>
             <td class="delete-contribution">
-              <a href="{{ url('main:view_fxosprice', contribution.pk) }}" class="btn btn-default btn-xs">Details</a>
+              <a href="{{ url('main:view_contribution', contribution.pk) }}"
+                 class="btn btn-default btn-xs">Details</a>
```

Comment:
The detailed view page has all the links together. Personally, I agree that we could add `edit` and `delete` here together with the `clone` button, instead of having the `details` button. But since this is not relevant to this PR, lets open either a new issue or a PR tackling what you are describing.